content/actions/creating-actions/setting-exit-codes-for-actions.md

```diff
@@ -48,3 +48,187 @@ fi
 ```
 
 For more information, see "[AUTOTITLE](/actions/creating-actions/creating-a-docker-container-action)."
+
+## Handling exit codes in composite actions
```

Comment:
Would you be able to shorten this whole section to a relevant example that explains how a non-zero exit in the middle of a composite action will cause the remaining composite action to behave? So that it is structured more similarly to the previous two sections about Docker and JavaScript actions?

The audience for this article is people who are creating actions. Because of that, any content that is directed towards people who are writing workflows should be removed from this article. If you think the documentation in the [Using workflows](https://docs.github.com/en/actions/using-workflows) section of the docs is missing any of this information, you can definitely open an issue to propose adding it there. 👍